### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by ceycocey** [elishacloud/dxwrapper#325] [General]
Hello, I play a game that runs better on directx9 so I wanted to ask how do I uninstall directx12 on my win10 and install directx9?
greetings

**Comment by elishacloud** [comment]
DirectX9 and DirectX12 can coexist.  There is no reason to remove DirectX12, that will just break functionality for some games.  Most games play only on specific versions of DirectX and if you remove it then the game won't work at all.

**Reply by elishacloud** [reply]
Download and install the DirectX June 2010 Runtime from here to get the latest version of DirectX9:  https://www.microsoft.com/en-us/download/details.aspx?id=8109
